source/wp-content/themes/wporg-news-2021/sass/post/_content.scss

```diff
@@ -23,6 +23,11 @@
 	}
 }
 
+body.archive .site-content-container {
+	padding-left: var(--wp--custom--alignment--edge-spacing);
+	padding-right: var(--wp--custom--alignment--edge-spacing);
+}
```

Comment:
```suggestion
	padding-right: var(--wp--custom--alignment--edge-spacing);
	padding-bottom: var(--wp--custom--alignment--edge-spacing);
}
```

I think this would address the spacing issue when there's no pagination mentioned [here](https://github.com/WordPress/wporg-news-2021/pull/278#issuecomment-1028066128).

Reply:
(I could also add this in a separate PR once this one is merged.)

Reply:
I was just testing it 🙂  It doe fix the issue, and adds more space under the pagination which better matches the design too

| Before | After | Design |
|--------|------|-------|
| <img width="608" alt="before" src="https://user-images.githubusercontent.com/541093/152853296-9ecb8fbe-4ba9-4c4b-be4c-5dbe707db511.png"> | <img width="621" alt="after" src="https://user-images.githubusercontent.com/541093/152853298-4355669c-0241-475b-a6c4-884dc214445f.png"> | <img width="520" alt="design" src="https://user-images.githubusercontent.com/541093/152853291-44e82219-9d1e-4996-8249-f1325a2b1145.png"> |